dotcom-rendering/src/components/SlotBodyEnd.importable.tsx

```diff
@@ -89,6 +91,17 @@ const buildBrazeEpicConfig = (
 	};
 };
 
+function getIsSignedIn(signedInStatus: SignedInStatus): boolean | undefined {
+	switch (signedInStatus) {
+		case 'Pending':
+			return undefined;
+		case 'SignedIn':
+			return true;
+		case 'NotSignedIn':
+			return false;
+	}
+}
```

Comment:
I saw this is duplicated in another file, would it be better to make this method exportable in the `identity.ts` file or `useSignedInStatus.ts`?